src/index.ts

```diff
@@ -133,6 +147,18 @@ async function putDirectory(
   }
 }
 
+async function cleanDirectory(ssh: node_ssh, remote: string, verbose = true) {
+  try {
+    await ssh.execCommand(`rm -rf ${remote}/*`);
```

Comment:
Unfortunately, this did not work with my windows server. I used `rmdir /s /q ${remote}` instead

Reply:
cool! Would still be nice to have this PR reviewed / merged. Think it's a useful feature...